docs/src/main/asciidoc/update-to-quarkus-3.adoc

```diff
@@ -0,0 +1,55 @@
+////
+This document is maintained in the main Quarkus repository, and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="update-projects-to-quarkus-3-automatically-howto"]
+= Update projects to Quarkus 3 automatically
+include::_attributes.adoc[]
+:categories: core
+:extension-status: "experimental"
+:summary: Update projects from Quarkus 2.13 and later to Quarkus 3.
+
+include::{includes}/extension-status.adoc[]
+
+As a developer, you can update your projects from Quarkus 2.13 and later to Quarkus 3 by running an update command.
```

Comment:
```suggestion
You can update your projects from Quarkus 2.x to Quarkus 3.x by running an update command.
```
a) do we need to be a developer to do this? can't we assume those reading this have developer skills?
 
b) why not just say Quarkus 2.x?

Reply:
Thanks. Updated. 
I had understood `quarkus update` worked only on 2.13 and later. 
@gsmet can you confirm that it works for Quarkus 2.7-based projects as well?